### PR TITLE
Re-execute block

### PR DESCRIPTION
**Pull request by zhangchiqing**

### CODE REVIEW

**Review by vishalchangrani — APPROVED** [review]
lgtm

File: cmd/util/cmd/rollback-executed-height/cmd/rollback-executed-height.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package remove
```
Comment (vishalchangrani) [inline cmd/util/cmd/rollback-executed-height/cmd/rollback-executed-height.go:1]: shouldn't this package be `cmd`?

File: cmd/util/cmd/rollback-executed-height/cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	flagDatadir string
+)
+
+// run with `./remove-execution-fork --datadir /var/flow/data/protocol`
```
Comment (vishalchangrani) [inline cmd/util/cmd/rollback-executed-height/cmd/root.go:15]:
```suggestion
// run with `./remove-execution-commit --datadir /var/flow/data/protocol`
```

File: cmd/util/cmd/rollback-executed-height/cmd/rollback-executed-height.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/module/metrics"
+	storagebadger "github.com/onflow/flow-go/storage/badger"
+)
+
+var (
+	flagHeight  uint64
+	flagDataDir string
+)
+
```
Comment (vishalchangrani) [inline cmd/util/cmd/rollback-executed-height/cmd/rollback-executed-height.go:17]:
```suggestion
// rolls back the execution height to the specified height
```

**Review by Kay-Zee — APPROVED** [review]

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**